live/root/usr/bin/info-cmdline-conf.sh

```diff
@@ -0,0 +1,25 @@
+#! /bin/sh
+
+set -e
+
+TARGET="${1:-/run/agama/cmdline.d/agama}"
```

Comment:
@imobachgs was already wondering if it should be use the .conf suffix (see https://github.com/agama-project/agama/pull/1970/files#r1958785104)